src/containers/FilmFrontpage/AllMoviesAlphabetically.tsx

```diff
@@ -148,20 +151,20 @@ const AllMoviesAlphabetically = () => {
           >
             <h2>{letter}</h2>
           </LetterHeading>
-          {movies.map((movie) => (
-            <StyledSafeLink
-              to={movie.contexts.filter((c) => c.contextType === "standard")[0]?.path ?? ""}
-              key={movie.id}
-            >
-              {movie.metaImage?.url && <MovieImage alt="" loading="lazy" sizes={"100px"} src={movie.metaImage.url} />}
-              <MovieTextWrapper>
-                <Heading textStyle="title.small" asChild consumeCss data-title="">
-                  <h3>{movie.title}</h3>
-                </Heading>
-                <Text textStyle="body.small">{movie.metaDescription}</Text>
-              </MovieTextWrapper>
-            </StyledSafeLink>
-          ))}
+          {movies.map((movie) => {
+            const context = movie.contexts.find((c) => c.rootId === FILM_ID);
```

Comment:
Kan vi droppe indententeringen her og heller finne contexten rett i `StyledSafeLink`?

Reply:
context brukes to ganger, så tenkte det blei enklere å lese dette.